lib/ext/heroku/command/addons.rb

```diff
@@ -174,7 +174,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      if addon.has_key? 'config_vars' && addon['config_vars']
```

Comment:
Probably not. I wasn't sure if the key is always present or not. I wanted to short circuit if the key isn't even there, but that means the second condition should be `addon['config_vars'].empty?` or something like that.